numba/tests/test_np_functions.py

```diff
@@ -2448,6 +2456,127 @@ def a_variations():
 
         self.assertIn("Cannot np.flip on UniTuple", str(raises.exception))
 
+    def test_logspace2_basic(self):
+
+        def Inputs():
+            #start, stop
+            yield 1, 60
+            yield -1, 60
+            yield -60, -1
+            yield -1, -60
+            yield 60, -1
+            yield 1.0, 60.0
+            yield -60.0, -1.0
+            yield -1.0, 60.0
+            yield 0.0, np.e
+            yield 0.0, np.pi
+            yield np.complex64(1), np.complex64(2)
+            yield np.complex64(2j), np.complex64(4j)
+            yield np.complex64(2), np.complex64(4j)
+            yield np.complex64(1 + 2j), np.complex64(3 + 4j)
+            yield np.complex64(1 - 2j), np.complex64(3 - 4j)
+            yield np.complex64(-1 + 2j), np.complex64(3 + 4j)
+
+        pyfunc = logspace2
+        cfunc = jit(nopython=True)(pyfunc)
+
+        for start, stop in Inputs():
+            self.assertAlmostEqual(pyfunc(start, stop).all(),
+                                   cfunc(start, stop).all())
```

Comment:
```suggestion
            np.testing.assert_allclose(pyfunc(start, stop), cfunc(start, stop))
```
If this code needs a looser testing regime, and it's all NumPy array outputs, I suggest using `np.testing.assert_allclose`.